src/components/MyHeader.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from "react";
+import "bootstrap/dist/css/bootstrap.css";
+import TodoList from "./TodoList";
+import UUID from "uuid";
+
+class MyHeader extends Component {
+  state = {
+    tasks: [],
+    nextId: UUID(),
+    input: "",
+    editItem: false,
+    completed: [],
+    completedTask: ""
+  };
+  handleInputChange = e => {
+    this.setState({
+      ...this.state,
+      input: e.target.value
+    });
+  };
+
+  addTodo = e => {
+    e.preventDefault();
+    /**
```

Comment:
maybe remove this comment or move it closer to state

Reply:
@theAndrewCline made that change